Twiter-HW-(Shna Rafeeq)/style.css

```diff
@@ -0,0 +1,165 @@
+*{
+    transition-timing-function: ease-in;
+    transition: 0.3s;
+  }
+
+body {
+	background: white;
+	font-size: 100%;
+	font-family: 'Times New Roman', serif; 
+	margin-top: 0;
+	margin-bottom: 0;
+}
+
+header {
+    position: sticky;
```

Comment:
don't really know why you needed to use a sticky header, maybe it's better to use fixed here 😅